app/Http/Controllers/RepositoryController.php

```diff
@@ -30,6 +30,10 @@ public function index(): Response
      */
     public function store(StoreRepositoryRequest $request): RedirectResponse
     {
+        if (!$request->user()) {
```

Comment:
why? not just handle on authorize?

Reply:
Php stan check

Reply:
This or ignore error, a good practice is to avoid check on authorize since we never go on request check something

Reply:
could we do it in other way i think it's not good to have smt like this.